### ISSUE TITLE
Visual Shader Editor : Cannot change color on Color Constant

### ISSUE THREAD

**Issue by lentsius-bark** [godotengine/godot#22906]
**Godot version:**
4c1a5d9

**OS/device including version:**
iMac macOS High Sierra 10.13.5

**Issue description:**
When a color constant is changed in Visual Shader Editor, the color does not update as it should and instead remains white.
This issue was fixed before, everything worked great and now came back few commits ago.

**Steps to reproduce:**
Make a new visual shader, open in visual shader editor, create a color const, hook it up with Albedo and try changing the default white value.

**Screenshot:**
![screen shot 2018-10-09 at 10 25 10](https://user-images.githubusercontent.com/38346332/46723374-fb0e6800-cc77-11e8-9d7e-1fedcc719973.png)

**Comment by fire** [comment]
@DualMatrix 

This line from 5f2f1089086ad64636efc2a4096e55e4483c927a breaks the colour constants.
```
diff --git a/editor/editor_properties.cpp b/editor/editor_properties.cpp
index a4170ea5b..8df49b451 100644
--- a/editor/editor_properties.cpp
+++ b/editor/editor_properties.cpp
@@ -1761,7 +1761,7 @@ void EditorPropertyColor::_color_changed(const Color &p_color) {
 
 void EditorPropertyColor::_popup_closed() {
 
-	emit_signal("property_changed", get_edited_property(), picker->get_pick_color(), true);
+	emit_signal("property_changed", get_edited_property(), picker->get_pick_color(), false);
 }
 
void EditorPropertyColor::_bind_methods() {
```

**Comment by akien-mga** [comment]
Fixed by #22915.
